kornia/augmentation/random_generator/_2d/resize.py

```diff
@@ -32,14 +33,12 @@ def __init__(self, resize_to: Union[int, Tuple[int, int]], side: str = "short")
         super().__init__()
         self.output_size = resize_to
         self.side = side
-        self.device: Optional[torch.device] = None
-        self.dtype: Optional[torch.dtype] = None
 
     def __repr__(self) -> str:
         repr = f"output_size={self.output_size}"
         return repr
 
-    def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
+    def make_samplers(self, device: Device, dtype: torch.dtype) -> None:
```

Comment:
let's use our `Dtype` class too

https://github.com/kornia/kornia/blob/815fa4a2eda76bbb4d9b3c024e0be3049b4fbcbd/kornia/constants.py#L60

Reply:
For this, will need to also add something like `DType.to_torch(self._dtype)`, and will break compatibility because can't stay using torch.dtype.